peachjam/views/generic_views.py

```diff
@@ -100,16 +103,20 @@ def add_facets(self, context):
                     a
                     for a in self.form.filter_queryset(
                         self.get_base_queryset(), exclude="authors"
-                    ).values_list("authors__name", flat=True)
+                    )
+                    .order_by()
+                    .values_list("authors__name", flat=True)
+                    .distinct()
                     if a
                 }
             )
 
         years = list(
             set(
```

Comment:
could we omit `set` since we have distinct below?

Reply:
Ah, yes, we can drop set everywhere then.